ember-cli-build.js

```diff
@@ -3,8 +3,14 @@
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 module.exports = function(defaults) {
   let app = new EmberApp(defaults, {
+    autoprefixer: {
+      enabled: true,
+      cascade: true,
+      sourcemap: true
+    },
     sassOptions: {
-      extension: 'scss'
+      extension: 'scss',
+      sourceMapEmbed: true
```

Comment:
Is config to exclude source maps in production builds taken care of?

Reply:
Is dat bad practice of zo?
Ik heb dat wel al gezien op andere sites soms ook naar buiten komt.
Persoonlijk lijkt me dit eigenlijk zelfs een feature (hierdoor kunnen we ook issues op productie sneller troubleshooten).
https://css-tricks.com/should-i-use-source-maps-in-production/

Reply:
mja ... ik had er nog nooit zo over nagedacht ("laat debugging op prod toe"). Als we ze houden, kan het dan in een aparte file? Met aparte map-files geef je browsers nog de kans om de source-maps enkel in te laden als je je debugger opent (voor JS sourcemaps doen browsers dat denk ik ... weet niet zeker ivm CSS maps), als ze embedded zijn, moet je ze sowieso inladen ( :arrow_down: transfer speed :arrow_down: parsing performance)

Reply:
Den autoprefixer speelt niet graag met Sass sourcemaps, dat was een rubriek. Ik link even de resource [hier](https://github.com/kimroen/ember-cli-autoprefixer)

```js
  const envIsProduction = (process.env.EMBER_ENV === 'production');

  ...

  sassOptions: {
    sourceMapEmbed: !envIsProduction
  },
  autoprefixer: {
    enabled: true,
    cascade: true,
    sourcemap: !envIsProduction
  }
```

Reply:
Probleem is, als ik dat draaide, dan faalde alles bij de build ☝️